src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,17 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
     public static void main(String[] args) {
-
+        Injector injector = Injector.getInstance("mate.academy");
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        Movie movie = new Movie();
+        movie.setTitle("Shawshank redemption");
+        movie.setDescription("Prison drama");
+        movieService.add(movie);
+        System.out.println(movieService.get(1L));
```

Comment:
```suggestion
        System.out.println(movieService.get(movie.getId()));
```
id was assigned to this movie, you want to get specific movie - get id from it

Reply:
done